tests/installation/start_install.pm

```diff
@@ -90,6 +90,12 @@ sub run() {
 
         # confirm
         send_key $cmd{install};
+        if (check_var('VERSION', '15')) { # SLE 15 has unsigned file errors, workaround them - rbrown 04/07/2017
+            while (check_screen('sle-15-unsigned-file')) {
```

Comment:
Same applies here, as we can end up with infinite loop. Acceptable as workaround though.